recipe/meta.yaml

```diff
@@ -1,34 +1,30 @@
 {% set name = "tslearn" %}
-{% set version = "0.5.3.2" %}
-{% set sha256 = "acf4015b07207ad6a645ddcc8eec04d8431716b24db66a745ab29fa22f772820" %}
+{% set version = "0.6.3" %}
 
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: c855c0b784b42dc4c50f4eaf524ec7ed658037e86fc14af38ca3b66c75dd6adb
 
 build:
   number: 0
-  skip: true  # [py2k]
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: true  # [py>312 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython                                 # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
-    - {{ compiler('c') }}
+    - python
+    - numpy
   host:
     - python
     - pip
-    - cython
-    - numpy
+    - numpy {{ numpy }}
+    - setuptools
+    - wheel
   run:
     - python
     - {{ pin_compatible('numpy') }}
```

Comment:
Could you try with  `- {{ pin_compatible('numpy', upper_bound='3') }}` instead?
To support numpy 2 until v3.

Reply:
Or better, just `numpy`. The numpy 2 versions have a run_exports. 
Only if you do get a good build with numpy 2.